CHANGELOG.md

```diff
@@ -1,19 +1,25 @@
 ## 10.4.1 (YYYY-MM-DD)
 
 ### Enhancements
-* None.
+* We now make a backup of a Realm file prior to any file format upgrade. The backup is retained for 3 months. Backups from before a file format upgrade allows for better analysis of any upgrade failure. We also restore a backup, if a) an attempt is made to open a realm file whith a "future" file format and b) a backup file exist that fits the current file format. The backup file is placed next to the real Realm file and is named `<fileName>.v<realmFileFormat>.backup.realm`.
+* The error message when the intial steps of opening a Realm file fails is now more descriptive.
 
 ### Fixes
 * [RealmApp] Client Reset errors now correctly forward the server error message. (Issue [#7363](https://github.com/realm/realm-java/issues/7363), since 10.0.0)
 * [RealmApp] All `AppException`s now correctly report the error message through `RuntimeException.getMessage()` instead of only through `AppException.getErrorMessage()`. 
+* [RealmApp] Proactively check the expiry time on the access token and refresh it before attempting to initiate a sync session. This prevents some error logs from appearing on the client such as: "ERROR: Connection[1]: Websocket: Expected HTTP response 101 Switching Protocols, but received: HTTP/1.1 401 Unauthorized" (RCORE-473, since v10.0.0).
+* Fix a race condition which could result in a skipping notifications failing to skip if several commits using notification skipping were made in succession (since v7.0.0).
```

Comment:
🤪 I am not sure I completely get this.

Reply:
Copy paste...I thought I understood it, but on a second read...I don't either 🙈

Reply:
I just removed it. Not sure there is any user facing behaviour we can describe this with that would make sense, nor do we have any open issues that is going to be fixed by it.